src/System.IO.Pipes/src/System/IO/Pipes/PipeStream.Unix.cs

```diff
@@ -4,21 +4,42 @@
 using Microsoft.Win32.SafeHandles;
 using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
-using System.Diagnostics.Contracts;
 using System.Runtime.InteropServices;
 using System.Security;
-using System.Threading;
-using System.Threading.Tasks;
 
 namespace System.IO.Pipes
 {
     public abstract partial class PipeStream : Stream
     {
+        private const string PipeDirectoryPath = "/tmp/";
+
+        internal static string GetPipePath(string serverName, string pipeName)
```

Comment:
You mean serverName?  Since we don't support cross-machine pipes in this implementation, I check whether serverName is either "." or the value returned by gethostname, and if it's neither, it throws a PlatformNotSupportedException.